etc/notes/CHANGES_5.0.0.md

```diff
@@ -1,12 +1,43 @@
-# Changes in v5
-
-## TOC
-
-- TODO
+# Changes in the MongoDB Node.js Driver v5
+
+* [About](#about)
+* [Changes](#changes)
+  + [Optional callback support migrated to `mongodb-legacy`](#optional-callback-support-migrated-to--mongodb-legacy-)
```

Comment:
A lot of these links don't work I think the TOC generator isn't happy with backtics. We could either edit the headers or fix the link generation, I think this is minor.